tests/TestCommon/project.json

```diff
@@ -0,0 +1,9 @@
+{
+    "version": "1.0.0-*",
+    "dependencies": {
+        "NETStandard.Library": "1.5.0-rc2-*"
```

Comment:
TestCommon is a dependency of DotNet.Tests so I assume the same versions should be pinned.